### ISSUE TITLE
Tabs: Selected tab from More menu should be visible on the tabs list

### ISSUE THREAD

**Issue by inforrpunay** [infor-design/enterprise#6990]
**Is your feature request related to a problem or use case? Please describe.**
It is possible in our application that the user opened several programs and each program has its own tab. Having multiple tabs results to having the "N More" menu to the right to be able to select the other tabs that can no longer be displayed in the main tab list. When the user selects a program from the More menu, they want the tab for the program to be visible so that they can identify which program is currently active. 

**Describe the solution you'd like**
Display the selected item from the More menu to the main tab list to identify that that item is currently selected or active.

**Additional context**
Kindly see link for expected behavior
https://user-images.githubusercontent.com/55830499/203485219-fd1c923c-0ed2-4d1e-8e8e-1a21bbea28b5.mp4

And what application do you work on? Infor M3

**Comment by tmcconechy** [comment]
This might be tricky as the only way i can think to do it is change the design so the top toolbar is really only a scroller and you scroll over to the tab rather than selecting it. (maybe combined with a button)

**Comment by inforrpunay** [comment]
Hi @tmcconechy, I think scrolling to the tab would make the user experience more difficult because the user would need to scroll to be able to see the tab that they want. Clicking on the More menu feels faster. How about putting the selected tab from the menu as the first tab and hiding the current displayed tab on the rightmost?

**Comment by tmcconechy** [comment]
@inforrpunay maybe an idea, but we had talked about a scroller (with arrows) as a possible replacement. We can make it work. But will take this idea / and yours up with Design

**Comment by inforandy** [comment]
coming out of grooming, we are moving forward here with a small fix to elevate any tab a user selects out of the overflow list into the last spot on the bar and bump the previous last tab down into overflow. any future enhancements to tab overflow behavior will be revisited as part of bigger tab initiatives in the future

**Comment by tmcconechy** [comment]
**To Dev**

In order to see the correct behavior...

1. https://main-enterprise.demo.design.infor.com/components/tabs/example-index.html
2. Resize to a mobile
3. Select the last tab
4. See how it scrolls the tab (tab notes comes into view)

But on https://main-enterprise.demo.design.infor.com/components/tabs-module/example-index.html it does not.

**Ask**
- make it scrollable on swipe like it is in normal tabs
- add the "fade"
- make sure when selecting it also goes into...
- basically make it work the same
